.github/workflows/workflow.yaml

```diff
@@ -61,6 +77,18 @@ jobs:
           for dir in cc go rust; do
               cd "$dir"
               echo
+              if [ "${{ runner.os }}" == "macOS" ]; then
+                case "$dir" in
+                  # Python - https://github.com/tweag/rules_nixpkgs/issues/186
+                  # Rust - https://github.com/tweag/rules_nixpkgs/issues/187
+                  python | \
```

Comment:
python will never be the case in this loop, we can safely omit it

Reply:
I just wanted to make sure we don't have different logic in different places here.

Reply:
Fixed in the new version.